.github/workflows/dependency-review.yml

```
@@ -0,0 +1,17 @@
+name: 'Dependency Review'
+on: [pull_request]
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repository'
+        uses: actions/checkout@v4
+      - name: 'Dependency Review'
+        uses: actions/dependency-review-action@v4
+        with:
+          comment-summary-in-pr: always
```

```suggestion
          comment-summary-in-pr: on-failure
```
Avoid the everything is okay alarm: https://www.youtube.com/watch?v=7vIjBtdEQRE

Options: https://github.com/actions/dependency-review-action?tab=readme-ov-file#configuration-options